owncloudData/src/main/java/com/owncloud/android/data/files/db/OCFileEntity.kt

```diff
@@ -121,6 +123,7 @@ data class OCFileEntity(
                 name = cursor.getString(cursor.getColumnIndexOrThrow(FILE_NAME)),
                 treeEtag = cursor.getString(cursor.getColumnIndexOrThrow(FILE_TREE_ETAG)),
                 lastSyncDateForData = cursor.getLong(cursor.getColumnIndexOrThrow(FILE_LAST_SYNC_DATE_FOR_DATA)),
+                lastUsage = cursor.getLong(cursor.getColumnIndexOrThrow(FILE_LAST_USAGE)),
```

Comment:
Not needed, we are not going to migrate this field from the old database

Reply:
Done